kelim_shluvim.py

```diff
@@ -0,0 +1,12 @@
+import itertools
+
+
+def interleave(*iterables: iter):
+    """
+    THe function get iterables and merge them alternately
+    :param iterables: iterables objects to merge
+    :return: the organs of the merged list
+    """
+    for organs in itertools.zip_longest(*iterables, fillvalue=None):  # Go over the iterables organs by locations
```

Comment:
You should consider change organs with elements, I think it would be a better term.